pkgs/servers/web-apps/healthchecks/default.nix

```diff
@@ -57,13 +57,40 @@ py.pkgs.buildPythonApplication rec {
     if SECRET_KEY_FILE:
         with open(SECRET_KEY_FILE, "r") as file:
             SECRET_KEY = file.readline()
+    EMAIL_HOST_PASSWORD_FILE = os.getenv("EMAIL_HOST_PASSWORD_FILE")
+    if EMAIL_HOST_PASSWORD_FILE:
+        with open(EMAIL_HOST_PASSWORD_FILE, "r") as file:
+            EMAIL_HOST_PASSWORD = file.readline()
+  '';
+
+  # Thanks to linuxserver.io for this code block: https://github.com/linuxserver/docker-healthchecks/blob/master/root/etc/cont-init.d/30-config#L116
+  createSuperuser = writeText "create_superuser.py" ''
+    from django.contrib.auth.models import User;
+    from hc.accounts.views import _make_user;
+    import os
+    SUPERUSER_EMAIL = os.getenv("SUPERUSER_EMAIL")
+    SUPERUSER_PASSWORD_FILE = os.getenv("SUPERUSER_PASSWORD_FILE")
+    if SUPERUSER_PASSWORD_FILE:
+        with open(SUPERUSER_PASSWORD_FILE, "r") as file:
+            SUPERUSER_PASSWORD = file.readline()
+    if SUPERUSER_EMAIL and SUPERUSER_PASSWORD:
+        if User.objects.filter(email=SUPERUSER_EMAIL).count()==0:
```

Comment:
spaces around `missing`

Reply:
Do you mean to add spaces around `==`, like so?

```python
if User.objects.filter(email=SUPERUSER_EMAIL).count() == 0:
```

Reply:
Yes

Reply:
Done 👍🏻